src/Config/Services.php

```diff
@@ -31,7 +32,12 @@ public static function renderer(?string $viewPath = null, ?ViewConfig $config =
             return static::getSharedInstance('renderer', $viewPath, $config);
         }
 
-        $viewPath = $viewPath ?: config('Paths')->viewDirectory;
+        if ($viewPath === null && ENVIRONMENT === 'testing') {
+            $viewPath = SUPPORTPATH . 'Views';
+        } else {
+            $viewPath = $viewPath ?: (new Paths())->viewDirectory;
```

Comment:
Please, remove all the changes and only leave this:
```php
$viewPath = $viewPath ?: ((new Optimize())->configCacheEnabled ?
    (new Paths())->viewDirectory :
    config('Paths')->viewDirectory
);
```

Reply:
It looks good. I understand the tests will work. A potential error may occur again
Is this a temporary fix? What happens if optimization is enabled (or another) in the future?

Reply:
This is the target solution. Using `config('Paths')->viewDirectory` is needed to be able to set a custom view path in the test environment. If config caching is disabled, you can safely use it.

I don't think there will be any problems with it.